src/main/java/sawkiewicz/task1/CartItem.java

```diff
@@ -0,0 +1,34 @@
+package sawkiewicz.task1;
+
+public class CartItem {
+    int quantity;
+    Item item;
+
+    public void setQuantity(int quantity) {
+        this.quantity = quantity;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public CartItem(int quantity, Item item) {
+        this.quantity = quantity;
+        this.item = item;
+    }
+
+    public CartItem() {
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
`equals` has a special meaning, the same as `hashCode` https://www.programcreek.com/2011/07/java-equals-and-hashcode-contract/

so, please don't use it here!

if you need some type of comparison implement `isSameAs` or in your case `hasSameQuantity`

Reply:
done